include/sharg/auxiliary.hpp

```diff
@@ -20,7 +20,11 @@
 namespace sharg
 {
 
-//!\brief Indicates whether application allows automatic update notifications by the sharg::argument_parser.
+/*!\brief Indicates whether application allows automatic update notifications by the sharg::argument_parser.
+ *
+ * \details
+ * \stableapi{Since version 1.0.}
```

Comment:
why not 1.0.`0` ?

Reply:
We won't change api on patch level

Reply:
This is the way in seqan3. The API should not be dependent on the patch version.